src/workerd/jsg/util.c++

```diff
@@ -570,29 +571,60 @@ kj::Array<kj::byte> asBytes(v8::Local<v8::ArrayBufferView> arrayBufferView) {
 }
 
 void recursivelyFreeze(v8::Local<v8::Context> context, v8::Local<v8::Value> value) {
```

Comment:
I think all uses of this go through the `jsg::Lock` now, or at least should be. Let's modify this to accept the `v8::Isolate*` as the first argument and have the call routed in through `jsg::Lock` pass in the `v8Isolate`.